api_spec.yaml

```diff
@@ -5394,9 +5938,74 @@ components:
           schema:
             $ref: '#/components/schemas/helloResponse'
 
-      
-
   schemas:
+    authRecipeUser:
```

Comment:
Please remove the old (recipe specific) user types)

Reply:
Done